auth/mockrandsource.go

```diff
@@ -0,0 +1,12 @@
+package auth
+
+import (
+	"log"
+	"math/rand"
+)
+
+// UseMathRand changes the random generator to math/rand, used only in testing environment
+func UseMathRand() {
```

Comment:
Could we have a test function like this instead:
```go
func Tokens(tokens... string) func() string {
	return func() string {
		if len(tokens) <= 0 {
			panic("no tokens left")
		}
		token := tokens[0]
		tokens = tokens[1:]
		return token
	}
}
```
and then add a variable to packages that need a token and then explicitly set tokens?
```go
func someTest() {
    tokenCreator = test.Tokens("firstToken", "secondToken")
    // revert tokenCreator with defer
    // execute testing method
}
```